packages/cloud-cognitive/config.js

```diff
@@ -17,13 +17,26 @@ export const getStorybookPrefix = (pkg, componentName) => {
     : storybookPrefixInternal;
 };
 
-export const getStorybookSlug = (pkg, componentName, scenario) => {
+/**
+ * A helper function to return the slug (structure path name reduced to lower
+ * case text and hyphens) which identifies story instances.
+ * @param {string} pkg The package settings instance.
+ * @param {string} folders The names of any sub-folders being used, or
+ * null/undefined if the component appears immediately within the category. The
+ * folders should be specified as a slug (all lower case, with hyphens rather
+ * than / to separate multiple folder levels).
+ * @param {string} componentName The name of the component.
+ * @param {string} scenario The scenario name, also as a slug.
+ * @returns
+ */
+export const getStorybookSlug = (pkg, folders, componentName, scenario) => {
```

Comment:
If we set a default parameter value for `folders` to `null` and have it be the last parameter of this function, we won't have to update all of these other files that use `getStorybookSlug`.

Something like...
```js
export const getStorybookSlug = (pkg, componentName, scenario, folders = null) => { ... }`
```